src/transformers/utils/peft_utils.py

```diff
@@ -91,7 +91,7 @@ def check_peft_version(min_version: str) -> None:
     if not is_peft_available():
         raise ValueError("PEFT is not installed. Please install it with `pip install peft`")
 
-    is_peft_version_compatible = version.parse(importlib.metadata.version("peft")) <= version.parse(min_version)
+    is_peft_version_compatible = version.parse(importlib.metadata.version("peft")) > version.parse(min_version)
```

Comment:
the `min_version` should be compatible. Using `> min_version` looks strange. We can `min_version` to use `>=` if necessary.